<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CellPreprocessWidget</class>
 <widget class="QWidget" name="CellPreprocessWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="GACLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QSlider" name="RadiusSlider">
      <property name="toolTip">
       <string>Specify the largest radius of a nuclei in terms 
of pixel spacing. 

Typically, a nucleus is in the range of 3-7 microns 
in radius.</string>
      </property>
      <property name="whatsThis">
       <string>Specify the largest radius of a nuclei in terms of pixel spacing. Typically, a nucleus is in the range of 3-7 microns in radius.</string>
      </property>
      <property name="maximum">
       <number>199</number>
      </property>
      <property name="value">
       <number>40</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksAbove</enum>
      </property>
      <property name="tickInterval">
       <number>5</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QDoubleSpinBox" name="RadiusSpinBox">
      <property name="toolTip">
       <string>Specify the largest radius of a nuclei in terms 
of pixel spacing. 

Typically, a nucleus is in the range of 3-7 microns 
in radius.</string>
      </property>
      <property name="whatsThis">
       <string>Specify the largest radius of a nuclei in terms of pixel spacing. Typically, a nucleus is in the range of 3-7 microns in radius.</string>
      </property>
      <property name="maximum">
       <double>20.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>4.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="RadiusLabel">
      <property name="toolTip">
       <string>Specify the largest radius of a nuclei in terms 
of pixel spacing. 

Typically, a nucleus is in the range of 3-7 microns 
in radius.</string>
      </property>
      <property name="whatsThis">
       <string>Specify the largest radius of a nuclei in terms of pixel spacing. Typically, a nucleus is in the range of 3-7 microns in radius.</string>
      </property>
      <property name="text">
       <string>CellRadius</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_104">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>90</y>
     <width>250</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="GlobalResetButton">
      <property name="toolTip">
       <string>Reset the parameters for all stages to 
default values.</string>
      </property>
      <property name="whatsThis">
       <string>Reset the parameters for all stages to default values.</string>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="GlobalApplyButton">
      <property name="toolTip">
       <string>Executes the pipeline stages 
in sequence</string>
      </property>
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>153</y>
     <width>311</width>
     <height>77</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="allChRadioButton">
      <property name="text">
       <string>All Channels</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string>buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="singleChRadioButton">
      <property name="text">
       <string>SingleChannels</string>
      </property>
      <attribute name="buttonGroup">
       <string>buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="ChannelComboBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="modelColumn">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>Channel 1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Channel 2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Channel 3</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>RadiusSpinBox</tabstop>
  <tabstop>RadiusSlider</tabstop>
  <tabstop>GlobalResetButton</tabstop>
  <tabstop>GlobalApplyButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
